### DISCUSSION TITLE
Compilation Errors with VS2022

### DISCUSSION THREAD

**Discussion by nullcharb** [winsiderss/systeminformer#1292] [Questions / Answers] [answered]
I have installed VS2022 on my system and I am trying to compile sysinformer. I have used both the commandline and building the solution from VS2022, in both cases I am getting a lot of errors. 

I would appreciate if someone could confirm if I am missing anything.
<img width="1771" alt="Screen Shot 2022-06-20 at 7 11 35 am" src="https://user-images.githubusercontent.com/58240340/174500555-c86f0ecd-3165-4573-acb8-66beb2030af7.png">
<img width="1440" alt="Screen Shot 2022-06-20 at 7 16 29 am" src="https://user-images.githubusercontent.com/58240340/174500572-ac4a759e-96b5-4018-bea7-776709604e8d.png">

**Comment by AltF5** [comment, marked as answer]
Odd errors. Can you confirm if you installed the latest Windows SDK and when first opening the project (redownload and see) if is not asking you to upgrade the SDK, which while could work, if you have the latest installed it should compile perfectly fine..

Make sure you have selected Debug | x64.

If still facing problems try VS2022 preview which worked very well, out of the box, for me

**Comment by nullcharb** [comment]
Ok so the SDK installation did it. There are still some errors with building the setup, which I will check later.

Thanks for your assistance.

![image](https://user-images.githubusercontent.com/58240340/174503097-8729f77a-8673-4276-a724-97d03941dd66.png)

**Reply by jxy-s** [reply]
This is due to the migration that is in progress. The source directories were renamed. We're still working on it: https://github.com/winsiderss/systeminformer/issues/1286
